test/mocks/config/cspReportUriImpl.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+
+module.exports = {
+	reportOnly: true,
+	impl: function (req) {
+		return 'http://www.example.com?token=sometoken';
+	},
+	policy: {
+		"default-src": "*"
+	}
+};
```

Comment:
Being consistent w/ the other files, but I went ahead and added a new line char anyways.